packages/lib-classifier/src/store/WorkflowStepStore/WorkflowStepStore.spec.js

```diff
@@ -363,13 +363,13 @@ describe('Model > WorkflowStepStore', function () {
       expect(rootStore.workflowSteps.shouldWeShowDoneAndTalkButton).to.be.false()
     })
 
-    it('should return false if configured to show the summary', async function () {
+    it('should return true if configured to show the summary', async function () {
```

Comment:
Maybe the wording here should be more like 'should ignore workflow configuration' to make it easier to understand what the code's intended to do? I'm not sure.